shared-data/js/types.ts

```diff
@@ -414,13 +418,28 @@ export interface CutoutFixture {
   height: number
 }
 
-type AreaType =
+type cuoutFixtureOmit = Omit<CutoutFixture, 'id' | 'providesAddressableAreas'>
+
+export interface FakeCutoutFixture extends cuoutFixtureOmit {
```

Comment:
```suggestion
export interface FakeCutoutFixture extends Omit<CutoutFixture, 'id' | 'providesAddressableAreas'> {
```
personal preference but imo this (and the same below and in other modules) is more readable because it's right there